File: examples/ios-objc/RudderTyperExample/ViewController.m
//
//  ViewController.m
//  RudderTyperExample
//
//  Created by Satheesh Kannan on 08/08/24.
//

#import "ViewController.h"
#import <Rudder/Rudder.h>
#import "RSRudderTyperAnalytics.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)triggerEventAction:(UIButton *)sender {
    [RSRudderTyperAnalytics sampleTrackEventName];
}

@end

File: tests/e2e/ios-swift/RudderTyperSwiftExample/ViewController.swift
//
//  ViewController.swift
//  RudderTyperSwiftExample
//
//  Created by Satheesh Kannan on 12/08/24.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    @IBAction func triggerEventAction(_ button: UIButton) {
        RudderTyperAnalytics.sampleEvent1()
    }
}

